MultiVersions/Forge/main/gildedarmor/lang/zh_tw.json

```
@@ -0,0 +1,17 @@
+{
+  "item.gildedarmor.gilded_netherite_helmet": "鑲金獄髓頭盔",
+  "item.gildedarmor.gilded_netherite_chestplate": "鑲金獄髓胸甲",
+  "item.gildedarmor.gilded_netherite_leggings": "鑲金獄髓護腿",
+  "item.gildedarmor.gilded_netherite_boots": "鑲金獄髓靴子",
+  "item.gildedarmor.gilded_enderite_helmet": "鑲金終界髓頭盔",

```

既然原版將 `Netherite` 的 `Nether` 簡寫為 `獄`，
那麼將 `Enderite` 的 `Ender` 簡寫為 `終`，會不會比較好呢？
```suggestion
  "item.gildedarmor.gilded_enderite_helmet": "鑲金終界髓頭盔",

```